src/test/kotlin/com/terraformation/backend/accelerator/db/DeliverableStoreTest.kt

```diff
@@ -97,9 +98,9 @@ class DeliverableStoreTest : DatabaseTest(), RunsAsUser {
       val deliverableId3 = insertDeliverable(id = 3, moduleId = 2)
       insertDeliverableDocument(templateUrl = "https://example.com/")
 
-      insertCohortModule(cohortId1, moduleId1)
-      insertCohortModule(cohortId1, moduleId2)
-      insertCohortModule(cohortId2, moduleId2)
+      insertCohortModule(cohortId1, moduleId1, LocalDate.of(2024, 1, 1), LocalDate.of(2024, 1, 31))
```

Comment:
Would it make sense to add a default value for the end date in `insertCohortModule`, say 6 days after the start date so the module is a week long, just to simplify call sites like this where we don't care about the specific dates?